tests/lib3/test_constructor.py

```diff
@@ -222,6 +223,10 @@ def tzname(self, dt):
         def dst(self, dt):
             return datetime.timedelta(0)
 
+    class MyFullLoader(yaml.FullLoader):
+        def _get_state_keys_blacklist(self):
```

Comment:
I like this way of doing things. Good job.

I'm not sure why you decided to make this a "private" method though. The fact that it gets overridden in a subclass seems to contradict that privateness.

Thoughts?

Reply:
Yeah I guess you are right :D I wanted to indicate they should not be directly used by other classes, but I will just remove the `_`.

Reply:
Fixed!